cccatalog-api/cccatalog/urls.py

```diff
@@ -124,8 +124,8 @@
         ImageStats.as_view(),
         name='about-image'
     ),
-    path('link', CreateShortenedLink.as_view(), name='make-link'),
-    path('link/<str:path>', ResolveShortenedLink.as_view(), name='resolve'),
+    # path('link', CreateShortenedLink.as_view(), name='make-link'),
```

Comment:
The reason that the tests are failing is because these two endpoints have been commented out – rather than disable the endpoint entirely, for now, what we ought to do is hide the documentation for them to discourage further use. You can achieve this by setting the `swagger_schema = False` attribute on the right classes in `link_views.py` (see `site_views.Healthcheck` for an example).

I would like to shut down the link shortener views entirely but we have to make sure they're not in use anywhere first.

Reply:
Understood and fixed